packages/cardano-services/test/StakePoolSeach/__snapshots__/StakePoolSearchHttpService.test.ts.snap

```diff
@@ -3091,6 +3001,21 @@ Object {
       },
       "vrf": "3409a1bebeaa47e6d99e0748a99f65dee60b7f7e9a64dc865d52b4fb445b98ab",
     },
+  ],
+  "totalResultCount": 6,
+}
+`;
+
+exports[`StakePoolSearchHttpService healthy state /search search pools by multiple filters identifier & status  & pledgeMet filters pledgeMet true, status activating, and condition 1`] = `
+Object {
+  "pageResults": Array [],
```

Comment:
DB snapshot likely doesn't include results, and it's not practical to try and capture another snapshot with them included. Ensuring test coverage here is not part of this PR scope though, as it's present in _master_

Reply:
Yes, I saw there are multiple places with empty results related to StakePoolSearchHttpService in _master_, no actions need to be taken as part of this PR correct?

Reply:
Correct. I'll make a ticket for this to be looked at separately